sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeHealthcareEntities.md

```diff
@@ -0,0 +1,210 @@
+                        The patient had a cardiac catheterization in July of this year revealing total occlusion of the RCA and 50% left main disease ,\
+                        with a strong family history of coronary artery disease with a brother dying at the age of 52 from a myocardial infarction and \
+                        another brother who is status post coronary artery bypass grafting. The patient had a stress echocardiogram done on July , 2001 , \
+                        which showed no wall motion abnormalities , but this was a difficult study due to body habitus. The patient went for six minutes with \
+                        minimal ST depressions in the anterior lateral leads , thought due to fatigue and wrist pain , his anginal equivalent. Due to the patient's \
+                        increased symptoms and family history and history left main disease with total occasional of his RCA was referred for revascularization with open heart surgery.";
+
+    HealthcareOperation healthOperation = client.StartHealthcare(document);
+
+    await healthOperation.WaitForCompletionAsync();
+
+    RecognizeHealthcareEntitiesResultCollection results = healthOperation.Value;
+
+    Console.WriteLine($"Results of Azure Text Analytics \"Healthcare\" Model, version: \"{results.ModelVersion}\"");
+    Console.WriteLine("");
+
+    foreach (DocumentHealthcareResult result in results)
+    {
+           Console.WriteLine($"    Recognized the following {result.Entities.Count} healthcare entities:");
+
+            foreach (HealthcareEntity entity in result.Entities)
+            {
+                Console.WriteLine($"    Entity: {entity.Text}");
+                Console.WriteLine($"    Subcategory: {entity.Subcategory}");
+                Console.WriteLine($"    Offset: {entity.Offset}");
+                Console.WriteLine($"    Length: {entity.Length}");
+                Console.WriteLine($"    IsNegated: {entity.IsNegated}");
+                Console.WriteLine($"    Links:");
+
+                foreach (HealthcareEntityLink healthcareEntityLink in entity.Links)
+                {
+                    Console.WriteLine($"        ID: {healthcareEntityLink.Id}");
+                    Console.WriteLine($"        DataSource: {healthcareEntityLink.DataSource}");
+                }
+            }
+            Console.WriteLine("");
+    }
+}
+```
+
+## Recognizing healthcare entities in multiple documents
+
+To recognize healthcare entities in multiple documents, call `StartHealthcareBatchAsync` on an `IEnumerable` of strings.  The result is a Long Running operation of type `HealthcareOperation` which polls for the results from the API.
+
+```C# Snippet:TextAnalyticsSampleHealthcareBatchAsync
+    string document = @"RECORD #333582770390100 | MH | 85986313 | | 054351 | 2/14/2001 12:00:00 AM | CORONARY ARTERY DISEASE | Signed | DIS | \
```

Comment:
it will be better to add another document instead of using the same one twice. It might confuse the user into thinking that they need to send the document twice or something

Reply:
updated